doc/14_data_frames.md

```diff
@@ -42,20 +42,18 @@ Example Changing Cluster/Keyspace Level Properties
 val conf = new SparkConf()
   .set("ClusterOne/spark.cassandra.input.split.size_in_mb","32")
```

Comment:
I think it would be best if we had an implicit so you could just do 
SparkConf.setClusterConfig("ClusterName", OneOfOurConfigurationCaseClasses)

I really don't want to add more magic than we currently have in the configuration and we've already documented the previous pattern in a bunch of places.